src/PoolManager.sol

```diff
@@ -227,16 +227,16 @@ contract PoolManager is IPoolManager, Fees, NoDelegateCall, ERC6909Claims {
         );
 
         _accountPoolBalanceDelta(key, delta);
-        // the fee is on the input currency
 
+        // the fee is on the input currency
         unchecked {
             if (feeForProtocol > 0) {
                 protocolFeesAccrued[params.zeroForOne ? key.currency0 : key.currency1] += feeForProtocol;
             }
         }
 
         emit Swap(
-            id, msg.sender, delta.amount0(), delta.amount1(), state.sqrtPriceX96, state.liquidity, state.tick, swapFee
+            id, msg.sender, -delta.amount0(), -delta.amount1(), state.sqrtPriceX96, state.liquidity, state.tick, swapFee
```

Comment:
why is the events delta the opposite as the returned delta?

Reply:
was doing this to align it with amoutnSpecified - which we are now going to fix in #491